### PR TITLE
Feature/serial functions

### PR DESCRIPTION
**Pull request by jackatbancast**
Added Serial functions to view and set the serial in memory as appropriate.

### CODE REVIEW

File: xtralien/imports.py [hunk 1]
```diff
@@ -70,6 +71,28 @@ def array_to_csv(arr, fname, *args, **kwargs):
...
 
 
+def load_csv(fname, array=False, headers=None, *args, **kwargs):
+    with open(fname, 'r') as f:
+        # Check if the file contains headers
+        if headers is None:
+            
+        
+
+        reader = csv.DictReader(f, *args, **kwargs)
```
Comment (ChrisOssila453) [inline xtralien/imports.py]: Does this need to be further indented to fall under the if statement above?

File: xtralien/imports.py [hunk 2]
```diff
@@ -70,6 +71,28 @@ def array_to_csv(arr, fname, *args, **kwargs):
...
+        _dict = OrderedDict()
```
Comment (ChrisOssila453) [inline xtralien/imports.py]: Same here?

**Review by ChrisOssila453 — COMMENTED** [review]
Check!

File: xtralien/imports.py (hunk 1 above)
Comment (jackatbancast) [reply xtralien/imports.py]: This was a part of a duplicate function that is un-needed

File: xtralien/imports.py (hunk 2 above)
Comment (jackatbancast) [reply xtralien/imports.py]: This was a part of a duplicate function that is un-needed

**Comment by jackatbancast** [comment]
I've removed to offending function, it should be all okay now.

**Review by ChrisOssila453 — APPROVED** [review]

**Pull request merged**